packages/nx-prisma-e2e/tests/nx-prisma.spec.ts

```diff
@@ -1,39 +1,7 @@
-// import { checkFilesExist, ensureNxProject, readJson, runNxCommandAsync, uniq } from '@nrwl/nx-plugin/testing';
+// TODO: Schematic Test
 
 describe('nx-prisma e2e', () => {
   it('should pass', () => {
     expect(true).toBeTruthy();
   });
```

Comment:
Right we don't have any schematics for `nx-prisma` so this would be used fro those whenever those are made.